### ISSUE TITLE
Similar performance on increasing number of vision queries for Finetuning Free expts on LVIS

### ISSUE THREAD

**Issue by anishmadan23** [YifanXu74/MQ-Det#55]
Hi,
Thanks for the code. I tried running the fine-tuning free method on LVIS with 5 vision queries and that seemed to give similar performance to that reported in the paper. But on increasing the vision queries to 10, I see similar performance to 5 vision queries. For instance, with 5 vq's: 42.9 AP vs 10 vq's: 42.8 AP on the  LVIS minival test set using GLIP-L. Is this expected behavior? If so, I would think that adding 5 additional images do not substantially improve the quality of features used for prompting. Could you kindly tell if this is consistent with your findings as well? 

Sample Commands used for the above expts:
**10 Vision Queries**

`python -m torch.distributed.launch --nproc_per_node 8 tools/test_grounding_net.py  --config-file configs/pretrain/mq-glip-l.yaml
--additional_model_config configs/vision_query_5shot/lvis_minival_L.yaml 
VISION_QUERY.QUERY_BANK_PATH MODEL/lvis_query_10_pool7_sel_large.pth 
MODEL.WEIGHT MODEL/mq-glip-l 
TEST.IMS_PER_BATCH 8 OUTPUT_DIR results/lvis/test_minival_10_shot_large/ VISION_QUERY.MAX_QUERY_NUMBER 10 VISION_QUERY.NUM_QUERY_PER_CLASS 10 DATASETS.FEW_SHOT 10`

**5 Vision Queries**
`python -m torch.distributed.launch --nproc_per_node 8  tools/test_grounding_net.py
--config-file configs/pretrain/mq-glip-l.yaml 
--additional_model_config configs/vision_query_5shot/lvis_minival_L.yaml 
VISION_QUERY.QUERY_BANK_PATH MODEL/lvis_query_5_pool7_sel_large.pth 
MODEL.WEIGHT MODEL/mq-glip-l 
TEST.IMS_PER_BATCH 8 OUTPUT_DIR results/lvis/test_minival_5_shot_large/ VISION_QUERY.MAX_QUERY_NUMBER 5 VISION_QUERY.NUM_QUERY_PER_CLASS 5 DATASETS.FEW_SHOT 5`

**Comment by YifanXu74** [comment]
Hi @anishmadan23 ,

I have also noticed that in certain datasets, such as LVIS, once the number of vision queries surpasses a certain threshold, increasing them further does not consistently lead to improvements. This may due to performance saturation. Addressing this limitation might require more refined pretraining strategies in future research.
